src/lib/components/select/select.svelte

```diff
@@ -1,11 +1,12 @@
 <script lang="ts">
   export let id: string = null;
   export let value: string | boolean;
+  export let color: string = '';
```

Comment:
Do we feel like just allowing _any_ string is too permissive? Also, I think it might get purged from Tailwind in some cases. If this is just to switch it between a dark and light mode, you have a few options…

- Have `dark` boolean that defaults to `false`. (This is probably the best option.)
- Use a union type to round it down to allowed colors (e.g. `export let color: 'light' | 'dark` = 'light'`).

Like I said, I'd probably go with the former.